client/src/components/CrossCheck/CrossCheckCriteriaForm.tsx

```diff
@@ -95,8 +95,8 @@ export function CrossCheckCriteriaForm({
     }
   }
 
-  function changeFinalScore(value: number) {
-    setTotalPoints(value > 0 ? value : 0);
+  function changeFinalScore(value: number | null) {
+    setTotalPoints(value != null && value > 0 ? value : 0);
```

Comment:
```suggestion
    setTotalPoints(value && value > 0 ? value : 0);
```

Reply:
It can be done even simplier :)
```suggestion
   setTotalPoints(value ? value : 0);
```